images/win/scripts/Tests/Svn.Tests.ps1

```diff
@@ -0,0 +1,6 @@
+
+Describe "Svn" {
+    It "svn exists" {
+        "svn --version --quiet" | Should -ReturnZeroExitCode
```

Comment:
all these tests that consists of the single simple test case should be placed in https://github.com/actions/virtual-environments/blob/master/images/win/scripts/Tests/Tools.Tests.ps1